tests/test_workflow_pipeline.py

```diff
@@ -507,6 +514,34 @@ def test_set_up_bids_db_ignore_patterns(workflow: PipelineWorkflow, tmp_path: Pa
     assert pybids_ignore_patterns == workflow.pybids_ignore_patterns
 
 
+def test_set_up_bids_db_no_session(
+    workflow: PipelineWorkflow,
+    tmp_path: Path,
+    caplog: pytest.LogCaptureFixture,
+):
+    """Create a fake BIDS dataset with no session-level folders.
+
+    Check that the ignore pattern is not added to the BIDS layout.
+    """
+    dpath_pybids_db = tmp_path / "bids_db"
+    participant_id = "01"
+    session_id = FAKE_SESSION_ID
+
+    fids.create_fake_bids_dataset(
+        output_dir=workflow.layout.dpath_bids,
+        subjects=participant_id,
+        sessions=session_id,
+    )
```

Comment:
Wouldn't this create a BIDS dataset **with** `ses-{FAKE_SESSION_ID}` folders? Shouldn't it be this instead:

```python
    fids.create_fake_bids_dataset(
        output_dir=workflow.layout.dpath_bids,
        subjects=participant_id,
    )
```

Reply:
It's probably also better to put some data (e.g. anat) in the BIDS dataset, then check that the BIDS layout object returned by `set_up_bids_db` is not empty, e.g. 
```python
    bids_layout = workflow.set_up_bids_db(
        dpath_pybids_db=dpath_pybids_db,
        participant_id=participant_id,
        session_id=session_id,
    )
    assert len(bids_layout.get(extension=".nii.gz")) > 0
```